Minor human planet/spaceport descriptions update (#7621)
endless-sky/endless-sky: data/map.txt

	landscape land/city9
	description `Longjump is a mostly industrial world, with lower than normal gravity that makes it relatively easy to operate and transport heavy equipment. Hang-gliding is also a popular sport here, although for safety reasons it is prohibited anywhere near the spaceport.`
	description `	Longjump was the first planet discovered in the region of space known today as the Rim. Every year in the month of August, the Rim celebrates Han Sizer month, a whole month of activities commemorating the first explorer of the region. Wealthier citizens of the Rim celebrate by visiting as many Rim worlds as they can in the month, paralleling Han Sizer's month-long exploration of the region.`

> 	spaceport `The spaceport consists of four large towers, connected by bridges, with openings for hangars in the sides. The upper stories are for landing ships; business takes place down below, at street level.`

Reviewer: ```suggestion
	spaceport `The spaceport consists of four large towers connected by bridges, with openings for hangars in the sides. The upper stories are for landing ships; business takes place down below, at street level.`
```

	landscape land/mfield1
	description `Millrace is an overpopulated world, full of factories and manufacturing plants. It is one of several planets in this region of space where people trying to escape from the crowded squalor and polluted cities of Earth come looking for work. Most of the goods manufactured here end up being used by the Syndicate for constructing starships or driving industry on other worlds.`
	description `	The Syndicate is a fair employer: living conditions here may be squalid, but most workers are earning a living wage and some are able to save up for a new life on a more pleasant planet. The Syndicate trusts there will never be a shortage of new immigrants to replace them.`
	spaceport `This spaceport is ugly. It was designed not for the pleasure of sight, but for processing as many people through it as possible, without collapsing onto itself.`

> 	spaceport `	As the population of Millrace has grown, the spaceport has been continually expanded to support the increase in immigration; you can recognize the different sections, because in the newer ones the bare concrete walls are lighter and have accumulated less soot and grime. Most of the restaurants are of the greasy spoon variety.`

Reviewer: ```suggestion
	spaceport `	As the population of Millrace has grown, the spaceport has been continually expanded to support the increase in immigration; you can recognize the different sections because in the newer ones the bare concrete walls are lighter and have accumulated less soot and grime. Most of the restaurants are of the greasy spoon variety.`
```

> 	spaceport `	Beyond the spaceport, you can see endless rows of apartment buildings receding  into the horizon, each exactly the same in shape, color, and level of filthiness.`

Reviewer 2: This line feels a *tiny* bit overdone.
```suggestion
	spaceport `	Beyond the spaceport, you can see endless rows of apartment buildings receding into the horizon. Any variation in design or color from one to the next is well-hidden by the layers of filth.`
```

> 	spaceport `	Despite what seems to be the extreme starkness of the spaceport, you can see endless rows of apartment buildings going into the horizon, each exactly the same in shape, color, and filth.`

Reviewer: ```suggestion
	spaceport `	Beyond the spaceport, you can see endless rows of apartment buildings receding  into the horizon, each exactly the same in shape, color, and level of filthiness.`
```

	landscape land/valley6
	description `Solace is a fertile world that was already home to Earth-like vegetation and animals even before the first settlers arrived. It has now been inhabited for nearly half a millennium, and sprawling cities cover much of its surface.`
	description `	In addition to being one of the major metropolitan centers of the galactic South, Solace is best known as the home of Delta V Corporation, makers of plasma engines and rockets of various sorts.`

> 	spaceport `Solace's spaceport is located in a satellite town to one of its major cities. The town itself is an extensive industrial park, stretching out for some distance. Eventually, the warehouses and factories give way to valleys and farmlands.` 

Reviewer: ```suggestion
	spaceport `Solace's spaceport is located in a satellite town of one of its major cities. The town itself is an extensive industrial park, stretching out for some distance. Eventually the warehouses and factories give way to valleys and farmlands.` 
```

> 	spaceport `Solace's spaceport is located in a satellite town of one of its major cities. The town itself is an extensive industrial park, stretching out for some distance. Eventually the warehouses and factories give way to valleys and farmlands.` 
> 	spaceport `	The spaceport serves not only for incoming and outgoing commerce, but also as a testing ground for the next generation of Delta V technology. On your way around the port you stop to watch some tests of engines so powerful that the ground shakes beneath them.`

Reviewer 2: Generally when something like this grows to encroach on farmland, you  have very developed industrial things that simply add on beyond the previous one at the nearest available distance. Which means you go from industrial park to farmland very quickly, often the farmland starts metres beyond the fenceline of the last warehouse.
```suggestion
	spaceport `Solace's spaceport is located in a satellite town of one of its major cities. The town itself is an extensive industrial park, stretching out for some distance where the warehouses and factories abruptly give way to valleys and farmlands.` 
	spaceport `	The spaceport serves not only for incoming and outgoing commerce, but also as a testing ground for the next generation of Delta V technology. On your way around the port you stop to watch some tests of engines so powerful that the ground shakes beneath them.`
```

Author: I see your point, but I think they both mean the same thing in the end. I'll change it to something else that makes it more clear anyway.

	attributes frontier north "north pirate" pirate
	landscape land/water4
	description `Zenith is a cold and unpleasant world, where the fog seldom lifts and the sun is rarely seen, where much of the lowlands are flooded each day by the tide, and storms are unpredictable and fierce. It has, however, one major advantage as a place to settle: it is far enough away that the Republic makes no attempt to control it.`

> 	description `	Several villages have been founded near the equator, and in addition there are an unknown number of private holdings, ranging from underground bunkers to enormous concrete fortresses. Because the land is owned by no government, and anyone can build a dwelling without permission or paperwork as long as they are willing to fight off any other claimants, the total population of Zenith is unknown.`

Reviewer: Like many of the other comma changes, this change is unnecessary.

```suggestion
	description `	Several villages have been founded near the equator, and in addition there are an unknown number of private holdings, ranging from underground bunkers to enormous concrete fortresses. Because the land is owned by no government and anyone can build a dwelling without permission or paperwork as long as they are willing to fight off any other claimants, the total population of Zenith is unknown.`
```
